core/services/relay/evm/functions/config_poller_test.go

```diff
@@ -205,3 +206,27 @@ func setFunctionsConfig(t *testing.T, pluginConfig *functionsConfig.ReportingPlu
 		OffchainConfig:        offchainConfig,
 	}
 }
+
+func generateDefaultOCR2OnchainConfig(minValue *big.Int, maxValue *big.Int) []byte {
```

Comment:
If you pass a `t`, then you can fail the test without disrupting the whole suite with `panic`.
```suggestion
func generateDefaultOCR2OnchainConfig(t *testing.T, minValue *big.Int, maxValue *big.Int) []byte {
```